<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkerLogin</class>
 <widget class="QDialog" name="WorkerLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>774</width>
    <height>465</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>员工</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_6">
      <attribute name="title">
       <string>车辆信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QTableWidget" name="tableWidget_5">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>车辆编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>品牌</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>颜色</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>车辆状态</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>购入费用</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>所需押金</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>租金(/h)</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>其他</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="ACButton_3">
           <property name="text">
            <string>增加车辆</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item>
            <widget class="QPushButton" name="DCButton_2">
             <property name="text">
              <string>删除车辆</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="RCButton_2">
             <property name="text">
              <string>修改信息</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_14">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>车牌号：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_5">
               <property name="maxLength">
                <number>9</number>
               </property>
               <property name="placeholderText">
                <string>C20140001</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="styleSheet">
              <string notr="true">color: rgb(239, 41, 41);</string>
             </property>
             <property name="text">
              <string>                              默认操作的是当前选中的车辆</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_8">
      <attribute name="title">
       <string>客户信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QTableWidget" name="tableWidget_7">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>性别</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>年龄</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>信用评级</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>账户余额</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>其他</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item>
          <widget class="QPushButton" name="DUButton_2">
           <property name="text">
            <string>删除客户</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_14">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_18">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>客户名称：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_7">
               <property name="maxLength">
                <number>10</number>
               </property>
               <property name="placeholderText">
                <string>username</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="styleSheet">
              <string notr="true">color: rgb(239, 41, 41);</string>
             </property>
             <property name="text">
              <string>                              默认操作的是当前选中的客户</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_9">
      <attribute name="title">
       <string>租车信息</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="tableWidget_8">
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>true</bool>
         </attribute>
         <column>
          <property name="text">
           <string>订单编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>员工编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>用户名</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>车辆编号</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>开始时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>结束时间</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>费用</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>违罚款</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>总费用</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>车辆使用情况评定</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_17">
             <item>
              <widget class="QRadioButton" name="radioButton_8">
               <property name="text">
                <string>租车时间段</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_6">
               <property name="text">
                <string>车辆编号</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_5">
               <property name="text">
                <string>只看请求</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="radioButton_7">
               <property name="text">
                <string>用户名</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QDateTimeEdit" name="TimeEdit1_2">
               <property name="displayFormat">
                <string>yyyy/M/d hh:mm:ss</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>～</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDateTimeEdit" name="TimeEdit2_2">
               <property name="displayFormat">
                <string>yyyy/M/d hh:mm:ss</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLineEdit" name="lineEdit_8">
               <property name="maxLength">
                <number>20</number>
               </property>
               <property name="placeholderText">
                <string>车辆编号/用户名</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="RentOkButton_2">
               <property name="text">
                <string>查询</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>订单编号:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lineEdit_order">
               <property name="maxLength">
                <number>12</number>
               </property>
               <property name="placeholderText">
                <string>T×××××</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>车辆状况:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <item>
                <property name="text">
                 <string>A</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>B</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>C</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>D</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>违罚款：</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="spinBox">
               <property name="maximum">
                <number>100000</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QPushButton" name="DealButton">
             <property name="text">
              <string>处理请求</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="widget_2">
      <attribute name="title">
       <string>我的主页</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>员工编号：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_num">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>员工名：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_name">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_26">
         <item>
          <widget class="QLabel" name="label_23">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>性别：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_sex">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_27">
         <item>
          <widget class="QLabel" name="label_24">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>年龄:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_age">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_28">
         <item>
          <widget class="QLabel" name="label_26">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>入职时间：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Ctime">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_29">
         <item>
          <widget class="QLabel" name="label_25">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>所完成交易数：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_tnum">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_31">
         <item>
          <widget class="QLabel" name="label_27">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>身份：</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_Ide">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_32">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>基本工资(/月):</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_wage">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_30">
         <item>
          <widget class="QPushButton" name="pushButton_info">
           <property name="text">
            <string>修改个人信息</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_pass">
           <property name="text">
            <string>修改密码</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_19">
     <item>
      <widget class="QPushButton" name="SlashButton">
       <property name="text">
        <string>刷新</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExitButton">
       <property name="text">
        <string>Logout</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
